src/Footer .js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Footer = props => {
+  return (
+    <div className="footer-app">
+      <ul>
+        {props.footer.map((element, i) => (
+          <li key={i}>{element}</li>
```

Comment:
It is best to try and avoid using the array index as they key, take a look at the react docs here https://legacy.reactjs.org/docs/lists-and-keys.html#keys

They do a better explanation than I ever could